src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -56,6 +56,11 @@ const addValidUser = async () => {
   const nextButton = await getNextButton();
   await waitFor(() => user.type(enterUserName, 'best'));
   await waitFor(() => expect(enterUserName).toHaveValue('best'));
+  const enterSshKey = screen.getByRole('textbox', {
+    name: /public SSH key/i,
+  });
```

Comment:
Very nit picky, but let's use async "findBy" queries in the tests to handle possible delays in rendering.

```suggestion
  const enterSshKey = await screen.findByRole('textbox', {
    name: /public SSH key/i,
  });
```